sim/druid/items.go

```diff
@@ -31,6 +32,21 @@ func init() {
 		druid.newBloodbarkCleaveItem(BloodBarkCrusher)
 	})
 
+	core.NewItemEffect(IdolOfTheDream, func(agent core.Agent) {
+		character := agent.GetCharacter()
+		character.OnSpellRegistered(func(spell *core.Spell) {
+			// In-game increases the weapon damage from 300% to 304%
+			if spell.SpellCode == SpellCode_DruidShred {
+				spell.DamageMultiplier += 2 * .02
+			}
+
+			// TODO: Also boosts the damage of swipe by 2%
+			// if spell.SpellCode == SpellCode_DruidSwipe {
+			// 	spell.DamageMultiplier *= 1.02
+			// }
+		})
+	})
```

Comment:
After some testing the idol seems to work the following for Shred:
1. The added flat damage is increased by 2%.
2. The base multiplier (225%) is increased by 2%, then the additive 75% from SoD are then added on top.
Because of that it needs to happen in Shred code after all.